docs/en/connector-v2/source/Hbase.md

```diff
@@ -17,75 +17,80 @@ Read data from Apache Hbase.
 
 ## Options
 
-|        name        |  type  | required | default value |
-|--------------------|--------|----------|---------------|
-| zookeeper_quorum   | string | yes      | -             |
-| table              | string | yes      | -             |
-| query_columns      | list   | yes      | -             |
-| schema             | config | yes      | -             |
-| hbase_extra_config | string | no       | -             |
-| common-options     |        | no       | -             |
+|        Name        |  Type   | Required | Default |
+|--------------------|---------|----------|---------|
+| zookeeper_quorum   | string  | Yes      | -       |
+| table              | string  | Yes      | -       |
+| schema             | config  | Yes      | -       |
+| hbase_extra_config | string  | No       | -       |
+| caching            | int     | No       | -1      |
+| batch              | int     | No       | -1      |
+| cache_blocks       | boolean | No       | false   |
+| common-options     |         | No       | -       |
 
 ### zookeeper_quorum [string]
 
-The zookeeper cluster host of hbase, example: "hadoop001:2181,hadoop002:2181,hadoop003:2181"
+The zookeeper quorum for Hbase cluster hosts, e.g., "hadoop001:2181,hadoop002:2181,hadoop003:2181".
 
 ### table [string]
 
-The table name you want to write, example: "seatunnel"
-
-### query_columns [list]
-
-The column name which you want to query in the table. If you want to query the rowkey column, please set "rowkey" in query_columns.
-Other column format should be: columnFamily:columnName, example: ["rowkey", "columnFamily1:column1", "columnFamily1:column1", "columnFamily2:column1"]
```

Comment:
Why remove this feature?

Reply:
> Why remove this feature?

This feature was not implemented in previous versions, and query_columns was an invalid configuration. After this modification, the query columns use the schema configuration. It is redundant to set both schema and query_columns at the same time.

The columns configured in the schema are equivalent to the columns in the query
```
schema = {
            columns = [
                  {
                     name = rowkey
                     type = string
                  },
                  {
                     name = "info1:age"
                     type = int
                  },
                  {
                      name = "info1:name"
                      type = string
                  }
]
}
```

Reply:
cc @EricJoy2048 @TyrantLucifer @TaoZex @hailin0 from #6348

Reply:
+1 from my side.